src/FileSdIBase.php

```diff
@@ -25,6 +25,22 @@ class FileSdIBase
 {
     public $NomeFile = null;
     public $File = null;
+	/* Set and array of UTF BOM chars, just in case of need */
+    private $BOM = array("\xEF\xBB\xBF",
```

Comment:
A che serve questo elenco di BOM?
E perchè lasciare le chiamate a removeBom se poi la funzione removeBom non fa più nulla?

Reply:
L'elenco di BOM non serve a nulla e penso si possa tranquillamente rimuovere, anzi adesso lo rimuovo: è una modifica che avevo implementato in seguito alla ricezione di file "boomati" UTF-16.
La funzione removeBom() l'ho lasciata perchè non ho analizzato gli altri file. Essendo public, non so se qualche altra funzione/classe si aspetti o meno di trovare il metodo.